app/javascript/App/images/lightbulb.svg

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where is this file used?

If it is being used, we should consider replacing it with a FontAwesome icon instead as well.